### DISCUSSION TITLE
Mobile support (BOOKMARKLET)

### DISCUSSION THREAD

**Discussion by Are-You-a-Person** [ThioJoe/YT-Spammer-Purge#884] [Ideas]
This may be a lot of work, but I think it would be worth it to a lot of different people. First of all a Bookmarklet will allow mobile users to be able to use this. It can also simplify the starting up process by A LOT, which will definitely make more people interested. A Bookmarklet is a bookmarked piece of code, like JavaScript, and can run on most browsers.

Someone just needs to write JavaScript either to simplify setting up the api, or to make this project somewhat mobile. Then use a bookmark maker like this [https://caiorss.github.io/bookmarklet-maker/](url)

**Comment by Are-You-a-Person** [comment]
The link doesn’t seem to work for me so just look up on Google “Bookmarklet maker”

**Comment by Are-You-a-Person** [comment]
Here is the simplest way to set up Google cloud.
1. Go to: https://console.cloud.google.com/projectcreate
2. Agree to terms
3. Make unique project name
4. Click create
5. Go to: https://console.cloud.google.com/apis/library/youtube.googleapis.com
6. Click enable
7. Click create credentials
8. Follow these steps: https://youtu.be/c6ebWvay8dE?t=122

**Comment by TechStudent10** [comment]
Might be possible in the future.
